source/alloc/src/lib.rs

```diff
@@ -1,5 +1,420 @@
-#![no_std]
+#![cfg_attr(not(feature = "use-std"), no_std)]
 
-pub mod containers;
-pub mod heap;
-pub mod node;
+// pub mod containers;
+// pub mod heap;
+// pub mod node;
+
+extern crate alloc;
+
+pub mod fornow {
```

Comment:
I plan to flatten this down in sort of the old layout, I wanted to keep the old one around for comparison until it worked. I'll remove the old allocator impl and replace it with this module soon.